Presentation.Web/Controllers/API/UserController.cs

```diff
@@ -173,5 +187,46 @@ public override HttpResponseMessage Delete(int id, int organizationId = 0)
             // NOTE: Only exists to apply optional param for org id
             return base.Delete(id, organizationId);
         }
+
+        [HttpGet]
+        [Route("api/user/rightsholders")]
+        public HttpResponseMessage GetUsersWithRightsholderAccess()
+        {
+            return _userRightsService
+                .GetUsersAndOrganizationsWhereUserHasRightsholderAccess()
+                .Match(
+                    success => success
+                        .OrderBy(x => x.Item2.Id)
```

Comment:
blab...Item2?. Brug navngivne value tuples i stedet for.

`Result<(User user, Organization organization),OperationError>`